### PR TITLE
(MODULES-2709) Note 1.9.3 PS reuse unsupported

### PR DESCRIPTION
**Pull request by Iristyle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -383,8 +383,12 @@ For Puppet 4.2.2+ (and 3.8.2) we've lessened the possibility of the issue occurr
 
 If you are affected by this issue, a known workaround is to download the `.tar.gz` from the forge and use `puppet module install` using the file rather than directly from the forge.
 
+- Use of this module with the 3.8.x x86 version of Puppet is highly discouraged, though supported.  Normally, this module employs a technique to dramatically improve performance by reusing a PowerShell process to execute DSC related commands.  However, due to the Ruby 1.9.3 runtime used with the 3.8.x x86 version of Puppet, this technique must be disabled, resulting in at least a 2x slowdown.
+
```
Comment (ferventcoder) [inline README.md]: This should probably move up to Limitations, not known issues.

Comment (ferventcoder) [inline README.md]: We are not going to correct this behavior - that's where I see the distinction.

Comment (Iristyle) [inline README.md]: Thanks - I missed the distinction. PR updated.

**Comment by jpogran** [comment]
LGTM

**Pull request merged**